modules/configuration-api/src/main/java/org/apache/ignite/configuration/NamedListChange.java

```diff
@@ -82,7 +82,21 @@
     NamedListChange<Change> createOrUpdate(String key, Consumer<Change> valConsumer);
 
     /**
-     * Remove the value from named list configuration.
+     * Renames the existing value in the named list configuration.
+     *
+     * @param oldKey Key for the value to be updated.
+     * @param newKey New key for the same value.
+     * @return {@code this} for chaining.
+     *
+     * @throws NullPointerException If one of parameters is null.
+     * @throws IllegalArgumentException If an element with name {@code newKey} already exists, or an element with name
+     *      {@code oldKey} doesn't exits, or {@link #delete(String)} has been invoked with the either {@code newKey}
```

Comment:
You can if it was deleted in previous "transaction". Here I mean that you can't delete "B" and rename "A" to "B" at the same time, it would lead to confusion

Reply:
ok, maybe this moment should be reflected in the javadoc